components/top-nav/src/asAnchor.js

```diff
@@ -13,26 +11,7 @@ const asAnchor = (AnchorType) => {
       .isRequired,
   };
 
-  const StyledHoc = styled(Anchor)({
```

Comment:
OK - so - I must admit that when I first looked at this I got a bit confused.  That's mostly because I was thinking of the `asAnchor` HOC that's in the HOC package.  The fact that this HOC was given an identical name wasn't a great choice.  :grin:  Maybe it should have been `asTopNavAnchor`

The idea of having an HOC like this is to avoid code duplication.  Looking through the rest of the code we do now have some duplication between the two anchor atom components.  It would probably be best to have the common styles provided by this HOC, and then only add/override to get the different custom elements of styling as necessary in those individual components.

As it stands right now this file has become a null component that just passes through all the props - it's used only in the unit test file and *could* therefore be deleted.

IMHO it would be best to restore styling into this component and simplify the atoms to remove duplication.

Reply:
> asTopNavAnchor

yep